9/Dockerfile

```diff
@@ -55,7 +55,7 @@ RUN wget https://github.com/Arachni/arachni/releases/download/v1.5/arachni-1.5-0
 
 RUN mkdir -p /var/run/redis && \
     wget -q --no-check-certificate \
-      https://svn.wald.intevation.org/svn/openvas/trunk/tools/openvas-check-setup \
+    http://wald.intevation.org/svn/openvas/trunk/tools/openvas-check-setup \
```

Comment:
Mind bumping this to `https`? I'm getting a 302 when hitting that directly anyway.